### DISCUSSION TITLE
Tray icons to be made to show

### DISCUSSION THREAD

**Discussion by hartsa999** [valinet/ExplorerPatcher#2320] [Ideas]
Customizing notification area icons does not appear to work. Only show all and show none works.
When "always show all icons..." is not selected, selecting above "Show icon and notifications" for any app does not show the icon, even after pressing Ok and restarting Windows Explorer. And after restarting ExplorePatcher the selections alway go back to "Only show icons".
So this functionality is broken.

(Company computer with 22H2 version of Windows and not possible to update until IM so decides (probably in a couple of years).)

https://imgur.com/a/m19QFUY

**Comment by pyrates999** [comment]
duplicate of: https://github.com/valinet/ExplorerPatcher/issues/1454

Please close this one.
